### ISSUE TITLE
Getting new icons

### ISSUE THREAD

**Issue by reflex90** [Uranium2/Auto_Hekili#3]
I saw that you prepared something in utils.py for gettings icons with the classes_spell list.
` def scrap_and_save_icons():
    path = "config\\classes_spells.json"
    with open(path, "r") as outfile:
        dict_ = json.load(outfile)
...`

How can I run a update for these images?

**Comment by Uranium2** [comment]
This step was manual from what I recall.
I used the smic files in `config/Notes/90_Death_Knight_Blood.simc` of each class to determine the spell used by the class (with a regex). You might want to update them also from the official smic github `https://github.com/simulationcraft/simc/tree/dragonflight/profiles`. Or you can add your profile, but it won't make a lot of difference since I don't check the rotation, but only to list used spells.

If you want to adapt/add/delete spells from a class:
You will need to find the spell from the simc profile that is missing, and add the new image to its class/spec folder. You do not need to update manually `config/classes_spells.json`. It is a generated file from the code.
You can scrap for wowhead icons with the spell_id given in the simc profile. Or you can check here to get the icons  `https://github.com/Gethe/wow-ui-textures/tree/live/ICONS`

For example if Hunter have a new spell called `frost_strike`,  add to the Hunter folder the image called `img/Hunter_BeastMastery/frost_strike.jpg`.

To recap, update smic profiles files.
Add new images with good naming (and file extension) in class/spec folder

**Comment by reflex90** [comment]
Hi thank you for your answer.
Already got managed to get the new evoker spec into the program.
But now I`m stuck getting all the icons for the spells from the classes_spells.json.
I saw you got something in utils.py to scrap the icons from wowhead and I wanted to know how I use this code to get all icons?
Dont want to do it manually :)

**Comment by Uranium2** [comment]
I think you can try to import the function an run it, it will use the json file to get all the icons. (Forgot a lot about this code)

Try to :
- Delete the "config\\classes_spells.json" file
- Update simc files, 
- Launch python in interactive :
```python
from src.utils import scrap_and_save_icons

scrap_and_save_icons()
```

It will be very slow because it will go over ALL spells from all classes/specs. Maybe you can make the process faster by editing the json file and keep only your class/spec you want to update. And then don't forget to delete the json file so that the project regenerate it.

**Comment by reflex90** [comment]
Thank you, would work if my chrome browser is chrome92 :D

**Comment by Uranium2** [comment]
Made a tool similar to this, since Hekili project will stop in retail.
https://github.com/Uranium2/one_button_rotation_auto_trigger
The one button rotation from Blizzard is not as perfect as Hekili.
https://github.com/Uranium2/one_button_rotation_auto_trigger
